pandas/core/generic.py

```diff
@@ -1906,6 +1906,10 @@ def _slice(self, slobj, axis=0, kind=None):
         return result
 
     def _set_item(self, key, value):
```

Comment:
Is this a single-point-of-contact that all (most?  many?) setter methods go through?  i.e. will the various `loc.__setitem__` paths eventually wind through here?